docs/source/introduction/probabilistic_numerics.rst

```diff
@@ -0,0 +1,39 @@
+Probabilistic Numerics
+=======================
+
+Mathematical models used to explain and predict
+the behaviour of complex systems such as immune cells or the climate
+rely heavily on numerical methods. The exponential growth in
+available data and computing power has revolutionized the scale of such models. 
+In practice given finite computational resources, this introduces 
+*epistemic uncertainty* arising from not running numerical
+methods to convergence and from
+observations corrupted by noise.
+
+In a Nutshell
+****************
+
+`Probabilistic numerics (PN) <http://probabilistic-numerics.org/>`_ aims to quantify uncertainty arising from intractable or incomplete numerical computation and from stochastic input. This new paradigm which has emerged at the intersection of computer science and numerical analysis treats a *numerical problem* as one of *statistical inference* instead. [#]_ [#]_ 
+
+The probabilistic viewpoint provides a principled way to encode structural knowledge about a problem. By giving an explicit role to uncertainty from all sources, in particular from the computation itself, PN gives rise to new applications beyond the scope of classical methods. For example, it makes consistent propagation of uncertainty the entirety of a computational pipeline possible.
```

Comment:
Typo in last sentence: "it makes [...] **over** the entirety of a computational pipeline possible"